src/aliceVision/photometricStereo/photometricDataIO.cpp

```diff
@@ -0,0 +1,185 @@
+#include <aliceVision/image/io.hpp>
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+
+#include "photometricDataIO.hpp"
+
+void loadLightIntensities(const std::string& intFileName, const std::vector<int>& usedPictures, std::vector<std::array<float, 3>>& intList)
+{
+    std::stringstream stream;
+    std::string line;
+    float x,y,z;
+    
+    std::fstream intFile;
+    intFile.open(intFileName, std::ios::in);
+    
+    if (intFile.is_open())
```

Comment:
prefer early exit conditions
```suggestion
    if (!intFile.is_open())
    {
        ALICEVIOSION_LOG_ERROR(....);
        throw ....;
    }
```